### PR TITLE
Fixed conditional logic order

### PR DESCRIPTION
**Pull request by krystian15**
Now attributes show codes not values

### CODE REVIEW

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/app/component/ProductInformation/ProductInformation.container.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4798443&bid=15970347)

File: src/app/component/ProductInformation/ProductInformation.container.js [hunk 1]
```diff
@@ -23,22 +23,20 @@ export default class ProductInformationContainer extends PureComponent {
...
 
-            const valueIndexFromParameter = parameters[key];
-            if (valueIndexFromParameter) {
-                const { label } = attribute_options[valueIndexFromParameter];
+            if (attribute_value && attribute_options[attribute_value]) {
+                const { label } = attribute_options[attribute_value];
                 return { ...acc, [attribute_label]: label };
             }
 
+            if (attribute_value) return { ...acc, [attribute_label]: attribute_value };
```
Comment (alfredsgenkins) [inline src/app/component/ProductInformation/ProductInformation.container.js]: I think it is best to prefer the `<ProductAttributeValue>` component. It has logic for all attribute types.
This also applied to `<ProductCard>` - where we render an attribute (i.e. brand). It also must be changed to `<ProductAttributeValue>`.

Comment (alfredsgenkins) [reply src/app/component/ProductInformation/ProductInformation.container.js]: Some changes, however must be applied to `<ProductAttributeValue>` the props must be made optional.

**Comment by alfredsgenkins** [comment]
This was implemented in #502 and #505, thanks for raising the issue! :)

**Comment by alfredsgenkins** [comment]
Let me know if I miss-understood this PR. (reopen it). Please add more details in that case :)

**Pull request closed without merging**